### DISCUSSION TITLE
国内网络安装失败后点击卸载卡死

### DISCUSSION THREAD

**Discussion by xiaoxiudog** [1Panel-dev/1Panel#5970] [Q&A]
### 1Panel 版本

v1.10.13-lts

### 问题描述

国内网络安装失败后点击卸载卡死

### 重现步骤

国内网络安装失败后点击卸载卡死

### 期待的正确结果

希望添加点国内可用镜像源，国内网络安装失败后点击卸载不卡死

**Comment by wanghe-fit2cloud** [comment]
> 国内网络安装失败后点击卸载不卡死

可以进入 `日志审计` 页面查看一下系统日志。

目前，所有应用镜像均来自应用官方，我们暂时不计划添加国内镜像源。你可以进入 `容器` - `配置` 页面，手动添加镜像加速器，以解决镜像下载超时的问题。
